gitopsaddon/test-scenarios.sh

```
@@ -545,25 +627,27 @@ cleanup_scenario() {
             kubeconfig=$OCP_KUBECONFIG; argocd_ns="openshift-gitops"
         fi
 
-        # ArgoCD CR should be removed by pre-delete Job (wait up to 180s)
+        # ArgoCD CR should be removed by pre-delete Job (wait up to 120s, then force)
         local w=0
-        while [ "$w" -lt 180 ]; do
+        while [ "$w" -lt 120 ]; do
             KUBECONFIG=$kubeconfig kubectl get argocd acm-openshift-gitops -n "$argocd_ns" &>/dev/null 2>/dev/null || break
```

Shellcheck (0.11.0)

[error] Multiple redirections compete for stderr. Use cat, tee, or pass filenames instead.

(SC2261)